lib/amorail/entity.rb

```diff
@@ -1,8 +1,8 @@
 require 'active_model'
+require 'active_support/time'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.